modules/framework/galasa-parent/dev.galasa.framework.api.ras/src/main/java/dev/galasa/framework/api/ras/internal/routes/RunDetailsRoute.java

```diff
@@ -167,19 +167,20 @@ private void resetRun(String runName) throws InternalServletException {
    }
 
    private void cancelRun(String runName, String result) throws InternalServletException {
-      boolean isCanceled = false;
+      boolean isCancelled = false;
       if (!result.equalsIgnoreCase("cancelled")){
          ServletError error = new ServletError(GAL5046_UNABLE_TO_CANCEL_RUN_INVALID_RESULT, runName, result);
          throw new InternalServletException(error, HttpServletResponse.SC_BAD_REQUEST);
       }
+
       try {
-         // Cancelling a run currently works by deleting all its entries in the DSS
-         isCanceled = framework.getFrameworkRuns().delete(runName);
+         isCancelled = framework.getFrameworkRuns().markRunCancelled(runName);
```

Comment:
not clear what isCancelled means in this context.

Reply:
Added a comment in https://github.com/galasa-dev/galasa/pull/273/commits/9268879a4e549b67260cb2fef9cb163d6c0ac963